AmplifyPlugins/API/AWSAPICategoryPluginTests/Operation/OperationTestBase.swift

```diff
@@ -12,21 +12,23 @@ import XCTest
 
 class OperationTestBase: XCTestCase {
 
+    var apiPlugin: AWSAPIPlugin!
+
     /// There are no throwing methods in this, but in order to let subclasses have a predictable way
     /// of setting up plugins, this base class uses `setUpWithError` instead of `setUp`. (XCTest
     /// lifecycle normally invokes `setUp` after `setUpWithError`, which means that any state config
     /// done inside of a subclass' `setUpWithError` could be erased by this class' call to
     /// `Amplify.reset` from inside `setUp`.
```

Comment:
Make sure to adjust comment to reflect whatever our final state is

Reply:
Fixed